baybe/constraints/discrete.py

```diff
@@ -41,6 +43,17 @@ def get_invalid(self, data: pd.DataFrame) -> pd.Index:  # noqa: D102
         res = reduce(_valid_logic_combiners[self.combiner], satisfied)
         return data.index[res]
 
```

Comment:
`return pl.lit(True)` instead of `pass` to fix mypy errors

Reply:
I dont understand why you would even consider doing `pass`.
 the mypy issue prob stems from the fact that the function output is cleary defined as polars expression, hence you cannot simply use `pass` as that would return `None`
so everything seems right to me?

Reply:
Yes, you're right.